Adafruit_BNO055.h

```diff
@@ -279,7 +279,7 @@ class Adafruit_BNO055 : public Adafruit_Sensor
       VECTOR_GRAVITY       = BNO055_GRAVITY_DATA_X_LSB_ADDR
     } adafruit_vector_type_t;
 
-#ifdef ARDUINO_SAMD_ZERO
+#if defined (ARDUINO_SAMD_ZERO) && ! (ARDUINO_SAMD_FEATHER_M0)
```

Comment:
Is the problem relative to the boards or to the SAMD21 implementation?
In latter case, to cover all Zero derivatives, the condition should be instead
`#ifdef ARDUINO_ARCH_SAMD`